src/Stripe.net/Services/_base/AnyOf.cs

```diff
@@ -0,0 +1,63 @@
+namespace Stripe
+{
+    using System.Collections.Generic;
+
+    #pragma warning disable SA1649 // SA1649FileNameMustMatchTypeName
+    public interface IAnyOf
+    {
+        object GetValue();
+    }
+    #pragma warning restore SA1649 // SA1649FileNameMustMatchTypeName
+
+    public class AnyOf<T1, T2> : IAnyOf
```

Comment:
Could we add some documentation here that explains what `AnyOf`'s intended purpose is for and a few words about how it's powered by `implicit`?